src/CMakeLists.txt

```diff
@@ -92,6 +92,8 @@ SET(root_headers
   ${PROJECT_SOURCE_DIR}/include/podio/ROOTReader.h
   ${PROJECT_SOURCE_DIR}/include/podio/ROOTLegacyReader.h
   ${PROJECT_SOURCE_DIR}/include/podio/ROOTWriter.h
+  ${PROJECT_SOURCE_DIR}/include/podio/Reader.h
+  ${PROJECT_SOURCE_DIR}/include/podio/Writer.h
```

Comment:
I think these should go into a separate list of headers, distinct from the root ones? That should also make the selection.xml things work?

Reply:
Fixed